src/main/java/org/terasology/moduletestingenvironment/fixtures/EmptyWorldGenerator.java

```diff
@@ -0,0 +1,36 @@
+// Copyright 2020 The Terasology Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+package org.terasology.moduletestingenvironment.fixtures;
+
+import com.google.common.collect.Lists;
+import org.terasology.engine.SimpleUri;
+import org.terasology.registry.In;
+import org.terasology.world.generation.BaseFacetedWorldGenerator;
+import org.terasology.world.generation.WorldBuilder;
+import org.terasology.world.generator.RegisterWorldGenerator;
+import org.terasology.world.generator.plugin.WorldGeneratorPlugin;
+import org.terasology.world.generator.plugin.WorldGeneratorPluginLibrary;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@RegisterWorldGenerator(id = "empty", displayName = "empty")
+public class EmptyWorldGenerator extends BaseFacetedWorldGenerator {
```

Comment:
What is this supposed to be used for? Could we add a little bit of JavaDoc explaining the intended use case?